packaging/rpm/make-microshift-app-images-rpm.sh

```diff
@@ -48,7 +113,7 @@ This rpm creates a RO container storage for user applications, pull the app imag
 
 %prep
 
-if [ -d  %{imageStore} ] 
+if [ -d  %{imageStore} ]
 then
    sudo rm -rf  %{imageStore}
 fi
```

Comment:
As an FYI, this block fails consistently, both on main branch and here, with:

```
+ '[' -d /tmp/ushift-rpm-package//BUILDROOT/microshift-app-images-1-1.x86_64/home/jcope/.local/share/containers/storage/overlay-images ']'
+ sudo rm -rf /tmp/ushift-rpm-package//BUILDROOT/microshift-app-images-1-1.x86_64/home/jcope/.local/share/containers/storage/overlay-images
rm: cannot remove '/tmp/ushift-rpm-package//BUILDROOT/microshift-app-images-1-1.x86_64/home/jcope/.local/share/containers/storage/overlay-images/overlay': Device or resource busy
error: Bad exit status from /var/tmp/rpm-tmp.Y9aEP7 (%prep)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.Y9aEP7 (%prep)
```

I'll open an issue for this. cc @mangelajo

Reply:
Thanks for the review @copejon!

I dug around on this. Reason I missed it is because it doesn't show up on Fedora35 but does on RHEL8.5. I'm guessing its the rpmbuild versions (Fedora `RPM version 4.17.0`) vs. (RHEL `RPM version 4.14.3`) but I'm having to knock off some rust on RPM building so that's a guess.

- Here is the buildroot exit:
```
Binary file /home/brent/rpmbuild/BUILDROOT/microshift-app-images-1-1.x86_64/var/lib/containers/storage/overlay-images/libpod/bolt_state.db matches
Found '/home/brent/rpmbuild/BUILDROOT/microshift-app-images-1-1.x86_64' in installed files; aborting
error: Bad exit status from /var/tmp/rpm-tmp.S0yRBl (%install)
```

In the meantime, an option is to skip the postinstall buildroot check with `%define __arch_install_post %{nil}` to get things working in master until we figure out the buildroot issue or if it isn't urgent maybe someone else might have an idea. I'm also chasing down an issue with building some different images that aren't resolving dependancies that I haven't nailed down yet.

Ty!

Reply:
> As an FYI, this block fails consistently, both on main branch and here, with:
> 
> ```
> + '[' -d /tmp/ushift-rpm-package//BUILDROOT/microshift-app-images-1-1.x86_64/home/jcope/.local/share/containers/storage/overlay-images ']'
> + sudo rm -rf /tmp/ushift-rpm-package//BUILDROOT/microshift-app-images-1-1.x86_64/home/jcope/.local/share/containers/storage/overlay-images
> rm: cannot remove '/tmp/ushift-rpm-package//BUILDROOT/microshift-app-images-1-1.x86_64/home/jcope/.local/share/containers/storage/overlay-images/overlay': Device or resource busy
> error: Bad exit status from /var/tmp/rpm-tmp.Y9aEP7 (%prep)
> 
> 
> RPM build errors:
>     Bad exit status from /var/tmp/rpm-tmp.Y9aEP7 (%prep)
> ```
> 
> I'll open an issue for this. cc @mangelajo

If you run as sudo I think that error should get resolved.